src/main/java/hudson/scm/SubversionEventHandlerImpl.java

```diff
@@ -36,9 +36,12 @@
 
     protected final File baseDir;
 
-    public SubversionEventHandlerImpl(PrintStream out, File baseDir) {
+    protected final boolean quietOperation;
```

Comment:
I would rather suggest using private fields and the getter. Otherwise the field cannot be removed in a compatible way if the configuration structure changes

Reply:
Alright. It will be updated in the following commit.